app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/MainActivity.java

```diff
@@ -65,28 +60,6 @@ private void switchFragment(Fragment f) {
         transaction.commit();
     }
 
-    // We overload the switchFragment function to allow the user to customize the
-    // transition between two fragments on a switch if they want. The two functions
-    // have identical behavior outside of the animation.
-    private void switchFragment(Fragment f, int customAnimationIn, int customAnimationOut) {
```

Comment:
Can we keep the switch Fragment method? Just cuz it's nice to have and it's cool to say that we implemented fragment switching and transitions.